Dockerfile

```diff
@@ -18,7 +17,7 @@ RUN set -x \
 # Exec container
 ###############################
 
-FROM alpine:3.10
+FROM alpine:3.15
```

Comment:
I'm planning to change a base image to debian, but to minimize a diff, update it to the latest alpine image for now.